src/main/java/ch/wisv/areafiftylan/products/controller/TicketRestController.java

```diff
@@ -119,13 +119,32 @@ public TicketRestController(TicketService ticketService, OrderService orderServi
     @PreAuthorize("hasRole('ADMIN')")
     @PostMapping("/types")
     public ResponseEntity<?> addTicketType(@RequestBody @Validated TicketType type) {
-
         TicketType ticketType = ticketService.addTicketType(type);
-
         return createResponseEntity(HttpStatus.CREATED, "TicketType successfully added", ticketType);
     }
 
     @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping("/types")
+    public ResponseEntity<?> readTicketTypes() {
+        Collection<TicketType> ticketTypes = ticketService.getAllTicketTypes();
+        return createResponseEntity(HttpStatus.OK, "TicketTypes successfully retrieved.", ticketTypes);
```

Comment:
You're getting a collection here, so just return those. No need for a wrapping response entity